JavaAssignment/src/rashmi/ReverseMaxString.java

```diff
@@ -0,0 +1,39 @@
+package rashmi;
+
+//Reverse Max length String
+
+public class ReverseMaxString 
+{
+	String length(String[] inpArray)
+	{
```

Comment:
here rashmi 
Never give method name which Java's class default provide 

String class already have length method...
it might confused that you might override String class length method in your own